src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator is now working.");

```

Comment:
The message 'Excavator is now working.' could be more informative by specifying the type of machine, for example, 'Excavator machine is now working.' to provide clarity.